src/Courses/CourseDetailsTabs/TopicsTab.tsx

```diff
@@ -114,6 +117,12 @@ export const TopicsTab: React.FC<TopicsTabProps> = ({ course, setCourse }) => {
         createTopic(courseUnitContentId);
     };
 
+    const setUnitInCourse = (unit: UnitObject) => {
```

Comment:
I would call this `addUnitToCourse`. The `set` confused me, when really all it's doing is pushing a unit onto the stack.